packages/dev/core/src/Materials/GaussianSplatting/gaussianSplattingMaterial.ts

```diff
@@ -232,6 +232,7 @@ export class GaussianSplattingMaterial extends PushMaterial {
         }
 
         effect.setFloat2("focal", focal, focal);
+        scene.bindEyePosition(effect);

```

Comment:
I think you should add a comment, else someone may remove this line in the future, thinking it's not necessary...

Reply:
fixed.